assets/src/edit-story/elements/text/edit.js

```diff
@@ -144,7 +147,13 @@ function TextEdit({
   const initialState = useMemo(() => {
     const contentWithBreaks = (content || '')
       .split('\n')
-      .map((s) => `<p>${s}</p>`)
+      .map((s) => {
+        // @todo This logic is temporary and will change with selecting part + marking bold.
```

Comment:
Let's move into a util module right away. E.g. module with `contentHtmlToDraftMarkup` and `draftMarkupToContentHtml` or such.

Reply:
Added a function for it to the text utils -- I'm not certain how the actual logic will look like for partial selection so perhaps that can be improved and separated more if needed in a later PR addressing that.